libs/wire-api-federation/src/Wire/API/Federation/Error.hs

```diff
@@ -130,6 +131,19 @@ federationUnavailable err =
     "federation-not-available"
     ("Local federator not available: " <> LT.fromStrict err)
 
+-- FUTUREWORK: Add Domain where the error occured to the WAI error responses here
```

Comment:
If I'm not mistaken, the domain is added by `federationErrorToWai`, so you can just remove this comment (and the one below).

Reply:
Ah good point! I thought this wasn't done yet.